doc/intro.rst

```diff
@@ -41,25 +41,25 @@ Philosophy
-|**Pypi**           |``$ pip install py3status``    |                                     |
-+-------------------+-------------------------------+-------------------------------------+
-|**Void Linux**     |``$ xbps-install -S py3status``|                                     |
-+-------------------+-------------------------------+-------------------------------------+
++-------------------+----------------------------------------+----------------------------------------------------------------------------+
+|Distro             |Helpful Command                         |Useful Note                                                                 |
++===================+========================================+=============================================================================
+|**Arch Linux**     |``$ pacaur -S py3status``               |Stable updates. Official releases.                                          |
++                   +----------------------------------------+----------------------------------------------------------------------------+
+|                   |``$ pacaur -S py3status-git``           |Real-time updates from master branch.                                       |
++-------------------+----------------------------------------+----------------------------------------------------------------------------+
+|**Debian & Ubuntu**|``$ apt-get install py3status``         |Stable update. In testing and unstable, and soon in bstable backports.      |
++-------------------+----------------------------------------+----------------------------------------------------------------------------+
+|**Fedora**         |``$ dnf install py3status``             |                                                                            |
++-------------------+----------------------------------------+----------------------------------------------------------------------------+
+|**Gentoo Linux**   |``$ emerge -a py3status``               |                                                                            |
++-------------------+----------------------------------------+----------------------------------------------------------------------------+
+|**Pypi**           |``$ pip install py3status``             |                                                                            |
++-------------------+----------------------------------------+----------------------------------------------------------------------------+
+|**Void Linux**     |``$ xbps-install -S py3status``         |                                                                            |
++-------------------+----------------------------------------+----------------------------------------------------------------------------+
+|**NixOS**          |``$ nix-env -i python3.6-py3status-3.7``|Not global. See `README <https://github.com/ultrabug/py3status#nixos>`_     |
```

Comment:
Suggestion: Keep it simple. `Not global. See README.rst.` And fix the `bstable` typo.